ui/pages/confirmations/hooks/useCurrentConfirmation.ts

```diff
@@ -48,27 +51,28 @@ const useCurrentConfirmation = () => {
         ({ id: confirmId }) => confirmId === paramsTransactionId,
       );
     }
+
     if (!pendingConfirmation) {
       if (!latestPendingConfirmation) {
         setCurrentConfirmation(undefined);
         return;
       }
       pendingConfirmation = latestPendingConfirmation;
     }
+
     if (pendingConfirmation.id !== currentConfirmation?.id) {
       const unconfirmedTransaction =
         unconfirmedTransactions[pendingConfirmation.id];
       if (!unconfirmedTransactions) {
         setCurrentConfirmation(undefined);
         return;
       }
-      if (
-        pendingConfirmation.type !== ApprovalType.PersonalSign &&
-        pendingConfirmation.type !== ApprovalType.EthSignTypedData
-      ) {
+
+      if (!ConfirmationRedesignTypes.includes(pendingConfirmation.type)) {
```

Comment:
Instead of widening the type for `ConfirmationRedesignTypes` overall, what do you think about assigning `ConfirmationRedesignTypes` to a local variable here so that we can temporarily widen the type? Something like

``` typescript
const confirmationRedesignTypes: string[] = ConfirmationRedesignTypes;

if (!confirmationRedesignTypes.includes(pendingConfirmation.type)) {
```

(feel free to move the variable higher up)

Reply:
There's no type alteration necessary if we simply avoid `includes`. 

```suggestion
      if (!ConfirmationRedesignTypes.find((type) => type === pendingConfirmation.type)) {
```

The underlying cause of the error in this PR that `includes` doesn't handle things well when handed string literal types and string types together.

I agree with @mcmire on everything else, especially not widening `ConfirmationRedesignTypes`.

Reply:
Cool solutions and thanks for sharing @mcmire & @MajorLift! Makes sense and looks good to me